src/servers/exporter.ts

```diff
@@ -311,3 +312,103 @@ export const getBookingVersusRealWorldAssets = async () => {
     (lightning + bitcoin)
   ) // value in accounting
 }
+
+createGauge({
+  name: "totalPendingHtlcCount",
+  description: "How many pending HTLCs there are in the channels of the active nodes",
+  collect: async () => {
+    try {
+      const lndService = LndService()
+      if (lndService instanceof Error) throw lndService
```

Comment:
in theory you dont need to wrap this code in try/catch, service should return an error so you can return NaN instead throwing the error and return NaN in the catch (this applies to the new `createGauge` handlers)

```js
if (lndService instanceof Error) return NaN
```

Reply:
remove the try/catch wrapping here: https://github.com/GaloyMoney/galoy/pull/2057/commits/6402dd14ef3a8438b0297deca8b28c4c08b424df

Reply:
the map solution doesn't check if the `inbound` property is present in the `inboundOutboundBalance` array and throws an error if there is no inbound present.